desktop/cmp/form/FormField.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+import React, {Component} from 'react';
+import {elemFactory, HoistComponent} from '@xh/hoist/core';
+import {PropTypes as PT} from 'prop-types';
+import {HoistInput} from '@xh/hoist/cmp/form';
+import {formGroup, spinner} from '@xh/hoist/kit/blueprint';
+import {fragment, div, span} from '@xh/hoist/cmp/layout';
+import {throwIf} from '@xh/hoist/utils/js';
+import {isArray} from 'lodash';
+
+import './FormField.scss';
+
+/**
+ * Standardised wrapper around HoistInputs.
+ *
+ * Should receive a single HoistInput as a child element. If the HoistInput is bound
+ * to a model enhanced by `@FieldSupport`, FormField will automatically display a label,
+ * a required asterisk and validation state.
```

Comment:
I think a key thing to express in the doc here is that, when wrapping with a FormField, the inputs themselves don't specify their usual model/field. That's delegated to / controlled by the wrapper.  We have the nice warnings in place, so you will find that out pretty fast, but still seems like a key thing to know from reading the docs.

This also refers to the HoistInput being bound, which it will be, but probably clearer to say "If this component is bound"...